Whoaverse/Whoaverse/Views/Shared/_Layout.cshtml

```diff
@@ -114,6 +114,25 @@
 
     <link rel='shortcut icon' href="/favicon.ico" type="image/x-icon" />
 
+    <!-- For Chrome for Android: -->
+    <link rel="icon" sizes="192x192" href="touch-icon-192x192.png">
```

Comment:
I believe this is overriding the existing favicon.ico (the blue arrow) as you did not add a new favicon.ico file. This thread has some complaints about it: https://voat.co/v/Voat/comments/76885. 

The more modern way to do icons for Chrome for Android is to make a web app manifest file like this: http://updates.html5rocks.com/2014/11/Support-for-installable-web-apps-with-webapp-manifest-in-chrome-38-for-Android